src/Physics/ini_model_DR.f90

```diff
@@ -262,17 +262,30 @@ SUBROUTINE DR_basic_ini(DISC,EQN,MESH,BND)                 ! global variables
 
     CASE(33) ! ImposedSlipRateOnDRBoundary
         allocate( nuc_xx(DISC%Galerkin%nBndGP,MESH%Fault%nSide),                    &
-                  nuc_yy(DISC%Galerkin%nBndGP,MESH%Fault%nSide)                     )
-
+                  nuc_yy(DISC%Galerkin%nBndGP,MESH%Fault%nSide),                    &
+                  DISC%DynRup%Yoffeonset(DISC%Galerkin%nBndGP,MESH%Fault%nSide),    &
+                  DISC%DynRup%YoffeTS(DISC%Galerkin%nBndGP,MESH%Fault%nSide),       &
+                  DISC%DynRup%YoffeTR(DISC%Galerkin%nBndGP,MESH%Fault%nSide)        )
+ 
         ! Initialize w/ first-touch
         !$omp parallel do schedule(static)
         DO i=1,MESH%fault%nSide
             nuc_xx(:,i) = 0.0d0
             nuc_yy(:,i) = 0.0d0
+            DISC%DynRup%Yoffeonset(:,i) = 0.0d0
+            DISC%DynRup%YoffeTS(:,i) = 0.0d0
+            DISC%DynRup%YoffeTR(:,i) = 0.0d0
         END DO
 
         call c_interoperability_addFaultParameter("strike_slip" // c_null_char, nuc_xx)
         call c_interoperability_addFaultParameter("dip_slip" // c_null_char, nuc_yy)
+        call c_interoperability_addFaultParameter("rupture_onset" // c_null_char, DISC%DynRup%Yoffeonset)
+        call c_interoperability_addFaultParameter("acc_time" // c_null_char, DISC%DynRup%YoffeTS)
+        call c_interoperability_addFaultParameter("effective_rise_time" // c_null_char, DISC%DynRup%YoffeTR)
```

Comment:
YoffeTS -> YoffeAccelerationTime
YoffeTR -> YoffeEffectiveRiseTime

Reply:
I think TS and TR are more exact than YoffeAccelerationTime and YoffeEffectiveRiseTime, which are defined by the magical formula discussed above.